src/main/lrsql/system/lrs.clj

```diff
@@ -356,4 +356,11 @@
     (let [conn  (lrs-conn this)
           input (admin-stat-input/query-status-input params)]
       (jdbc/with-transaction [tx conn]
-        (admin-q/query-status backend tx input)))))
+        (admin-q/query-status backend tx input))))
+
+  adp/AdminLRSManager
+  (-delete-actor [this params]
+    (let [conn (lrs-conn this)
+          input (agent-input/delete-actor-input params)]
```

Comment:
I see, you use the raw params map as a shortcut and presumably its got the same name. Hrmm, so technically this pulls anything from params and jams it into a db input. If we're trying to be consistent I would destructure params into `actor-ifi`, pass that into the input fn, and put it BACK on a map. Seems like extra work but it also does get rid of anything extraneous and ensure consistent format.

Reply:
This will ensure proper worship of the cargo cult 😛

Reply:
volcano god: sacrificed to